File: java/using-webhooks/GetLatestContent.java
// DocSection: using_webhooks_get_latest_content
// Tip: Find more about Java SDKs at https://docs.kontent.ai/java
import kentico.kontent.delivery.*;

DeliveryOptions deliveryOptions = new DeliveryOptions();
deliveryOptions.setProjectId("<YOUR_PROJECT_ID>");
deliveryOptions.setWaitForLoadingNewContent(true);

DeliveryClient client = new DeliveryClient(deliveryOptions);

CompletionsStage<ContentItemResponse> item = client.getItem("my_article");
// EndDocSection

File: php/using-webhooks/GetLatestContent.php
// DocSection: using_webhooks_get_latest_content
<?php
// Tip: Find more about PHP SDKs at https://docs.kontent.ai/php

// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kentico\Kontent\Delivery\DeliveryClient;

$client = new DeliveryClient("<YOUR_PROJECT_ID>", null, true);

$item = client->getItem('my_article');
// EndDocSection